staff/lab/mod-printer

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+"""Remove or add printers to/from CUPS classes."""
+import argparse
+import subprocess
+import sys
+
+
+def modify_printer(args):
+    """Perform internal call to CUPS CLI to modify class(es)."""
+    # Determine which CUPS classes are available
+    cups_classes = subprocess.check_output(["lpstat", "-p"]).decode("utf-8").split("\n")
+    # Remove empty trailing newline
+    cups_classes.remove("")
+    for i in range(len(cups_classes)):
+        clazz = cups_classes[i]
+        # Output is of the form: printer <printername> is idle. <etc>
+        clazz = clazz[len("printer "):]
+        clazz = clazz[:clazz.index(" is ")]
+        cups_classes[i] = clazz
```

Comment:
This makes the code harder to read, since you're reusing the `cups_classes` variable to have two purposes and two different meanings.

I'd recommend making the first variable called `command_output` or something like that, and the second variable can be `cups_classes` (the actual list of class names).

Also recommend using a regex to parse the command output lines instead of this kind of string manipulation.